docs/corectl.md

```diff
@@ -13,9 +13,14 @@ corectl [flags]
 ### Options
```

Comment:
It says options here, but `corectl [flags]` in the instruction above, normalize?